src/compiler/types.ts

```diff
@@ -1780,12 +1863,30 @@ namespace ts {
         kind: SyntaxKind.ClassExpression;
     }
 
-    export interface ClassElement extends RealDeclaration {
+    export interface ClassElement extends DeclarationBase {

```

Comment:
Another nearly-3% slowdown for a combined 106.4% slowdown. On my machine compilation went from 46.1 &rarr; 47.7 &rarr; 49.0 seconds